mainframer.sh

```diff
@@ -67,7 +69,7 @@ function syncBeforeBuild {
 	echo "Sync local → remote machine..."
 	startTime=`date +%s`
 
-	COMMAND="rsync --archive --delete --relative= --compress-level=$LOCAL_COMPRESS_LEVEL "
+	COMMAND="rsync --archive --delete --rsync-path=\"mkdir -p $PROJECT_DIR_ON_REMOTE_MACHINE && rsync\" --compress-level=$LOCAL_COMPRESS_LEVEL "
```

Comment:
Ok, I also forgot to escape `$PROJECT_DIR_ON_REMOTE_MACHINE` and it'll fail in case if we decide to add spaces.